.github/workflows/check-cmdstan.yaml

```diff
@@ -0,0 +1,77 @@
+name: check-cmdstan
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    paths:
+      - '**.stan'
+    branches:
+      - main
+      - develop
+  schedule:
+    - cron: '5 4 * * 1'
+  pull_request:
+    paths:
```

Comment:
Some merge issues here?

Reply:
Yes...working between my Mac and PC gave me some weirdness. Will pull these out and rebase with main.